File: Logic Building/Basic Problems/Check Even or Odd/Even_odd.py
# Procigaral approach

# n = int(input("Enter Number: "))

# if(n % 2 == 0):
#     print(f"The Given number {n} is even")

# elif(n % 2 != 0):
#     print(f"The Given number {n} is odd")

# else:
#     print("Something Went Wrong")


# Functional Approach

def isEven(n):
    if n%2 ==0 :
        return True
    else:
        return False
    
m = int(input("Enter number: "))

print("The Given number is Even or Not (if true return true else false)",isEven(m))

File: Logic Building/Basic Problems/Check Even or Odd/Even_odd.c
#include <stdio.h>
#include <stdbool.h>


int iseven(int n);
int main()
{
    int number;
    scanf("%d", &number);

    // Procigral Approach
    // if (number%2 == 0)
    // {
    //     printf("The Given number is Even");
    // }
    // else if (number != 0)
    // {
    //     printf("The Given number is Odd");
    // }
    // else{
    //     printf("Something went Wrong!");

    // }

    // Functional Approach

    printf("if the output is true number is even else odd: %d",iseven(number));
    return 0;
}
int iseven(int n)
{
    if (n % 2 == 0)
    {
       return true;
    }
    else if (n % 2 != 0)
    {
      return false;
    }
    
} 